plugins/qlcplus_4.11.2/import.js

```diff
@@ -83,6 +84,98 @@ module.exports.import = function importQlcPlus(buffer, filename, authorName) {
     });
 };
 
+/**
+ * Try to extract (guessed) wheels from all channels / capabilities.
+ * @param {object} qlcPlusFixture The QLC+ fixture object.
+ * @returns {object|undefined} The OFL wheels object or undefined if there are no wheels.
+ */
+function getOflWheels(qlcPlusFixture) {
+  const wheels = {};
+
+  for (const channel of qlcPlusFixture.Channel || []) {
+    const channelName = channel.$.Name;
+    if (/wheel\b/i.test(channelName)) {
```

Comment:
```suggestion
    if (/wheel\b/i.test(channelName) && !/(rotation|index)/i.test(channelName) && channel.Group[0]._ !== `Speed`) {
```

This won't create wheels for `Gobo rotation` channels